PROTOCOL.md

```diff
@@ -265,19 +265,25 @@ The following is an example `metaData` action:
 <!-- TODO: forward references configuration options -->
 
 ### Add File and Remove File
-The `add` and `remove` actions are used to modify the data in a table by adding or removing individual data files respectively.
+The `add` and `remove` actions are used to modify the data in a table by adding or removing individual _logical files_ respectively.
 
-The path of a file acts as the primary key for the entry in the set of files.
-When an `add` action is encountered for a path that is already present in the table, statistics and other information from the latest version should replace that from any previous version.
-As such, additional statistics can be added for a path already present in the table by adding it again.
+Every _logical file_ of the table (referred to as just a _file_ going forward) is represented by a path to a data file, combined with a Deletion Vector (DV) that indicates which rows of the data file are no longer in the table. The path of the data file acts as the primary key for the entry in the set of files. Deletion Vectors are an optional feature. In tables that do not have this feature enabled, all Deletion Vectors are considered to be empty.
```

Comment:
```suggestion
Every _logical file_ of the table (referred to as just a _file_ going forward) is represented by a path to a data file, combined with an optional Deletion Vector (DV) that indicates which rows of the data file are no longer in the table. The path of the data file acts as the primary key for the entry in the set of files. 
```

Reply:
Maybe its also better to define as "physical data files" as a the combination of multiple types of files that represent a single "logical data file". Then henceforth we can use "physical data files" instead of "data file + DV" all over the place. This will make the doc easier to update later if we add more physical file type in the future.

Reply:
The idea here was to introduce that "empty DV" is a legal DV, so we don't constantly have to spell out "optional DV" in all the places, but just write DV and it's implied that it may be empty (or null).

As for "physical data files", I'm not sure what this is supposed to refer to. I don't think we should treat the DV a "physical data file". It's not even always a "physical file", since it can be inline. Data files are e.g. parquet files, or anything that actually carries table data, not just protocol-level annotations like the DV (file or inline) does.

Reply:
This "empty DV" is a "legal DV" is hardly intuitive. For a casual reader (even for me) this was not obvious. So i strongly suggest writing this better.

Reply:
Ok, I went through everything again and tried to be very explicit about usage of "logical file" or a specific primary key tuple where appropriate/necessary.